upup/pkg/fi/cloudup/awstasks/load_balancer_attachment.go

```diff
@@ -125,3 +126,33 @@ func (_ *LoadBalancerAttachment) RenderAWS(t *awsup.AWSAPITarget, a, e, changes
 	}
 	return nil
 }
+
+type terraformLoadBalancerAttachment struct {
+	ELB              *terraform.Literal `json:"elb"`
+	Instance         *terraform.Literal `json:"instance,omitempty"`
+	AutoscalingGroup *terraform.Literal `json:"autoscaling_group_name,omitempty"`
+}
+
+func (_ *LoadBalancerAttachment) RenderTerraform(t *terraform.TerraformTarget, a, e, changes *LoadBalancerAttachment) error {
+	tf := &terraformLoadBalancerAttachment{
+		ELB: e.LoadBalancer.TerraformLink(),
+	}
+
+	if e.AutoscalingGroup != nil && e.Instance == nil {
+		tf.AutoscalingGroup = e.AutoscalingGroup.TerraformLink()
+		return t.RenderResource("aws_autoscaling_attachment", *e.AutoscalingGroup.Name, tf)
+	} else if e.AutoscalingGroup == nil && e.Instance != nil {
+		tf.Instance = e.Instance.TerraformLink()
```

Comment:
(We might want to deprecate instance attachment, but not in this PR.  I don't think anyone uses it...)